### PR TITLE
(site/ls) bump rke to 1.5.8

### PR DESCRIPTION
**Pull request by cbarria**
Bump the rke client to version 1.5.8 to entire LS

### CODE REVIEW

**Review by jhoblitt — CHANGES_REQUESTED** [review]
The rke version can also be removed form `cluster/antu/role/rke.yaml`.

**Review by jhoblitt — APPROVED** [review]
LGTM.

File: spec/hosts/nodes/gaw01.ls.lsst.org_spec.rb
Comment (dtapiacl) [inline spec/hosts/nodes/gaw01.ls.lsst.org_spec.rb:1]: Question ?: How was the new checksum generated ? @cbarria

**Review by dtapiacl — APPROVED** [review]
LGTM.

File: spec/hosts/nodes/gaw01.ls.lsst.org_spec.rb
Comment (cbarria) [reply spec/hosts/nodes/gaw01.ls.lsst.org_spec.rb:1]: if you go to the project page on github, there is a sha256 checksum file in the lower part
https://github.com/rancher/rke/releases/tag/v1.5.8

**Pull request merged**